mava/networks/base.py

```diff
@@ -232,3 +233,73 @@ def __call__(
         eps_greedy_dist = MaskedEpsGreedyDistribution(q_values, eps, obs.action_mask)
 
         return hidden_state, eps_greedy_dist
+
+
+class QMixNetwork(nn.Module):
```

Comment:
I feel we should call this a mixing network, either `QMixingNetwork` or just `MixingNetwork` because at first glance it seems like the network we use in qmix for everything